packages/core/src/Stores/contract-trade-store.js

```diff
@@ -171,6 +174,11 @@ export default class ContractTradeStore extends BaseStore {
         const tick_update_timestamp = should_update_contract_barriers
             ? this.accumulator_contract_barriers_data.tick_update_timestamp
             : this.accumulator_barriers_data.tick_update_timestamp;
+        if (document.hidden) {
```

Comment:
`document.hidden` is true when browser tab is inactive. When browser tab is inactive, i'm debouncing the `setTimeout` by clearing it before each run because `Chrome` increases timers of each setTimeout in the queue to 1 sec when running in the background.
And I'm not using `debounce` by `lodash` here because `debounce` doesn't work inside `action` and I would need to refactor it all to use debounce, so I chose a simple `clearTimeout` instead which serves the same purpose just as well.